nebula/ui/components/MZRadioDelegate.qml

```diff
@@ -114,8 +115,15 @@ RadioDelegate {
     MZRadioButtonLabel {
         id: radioButtonLabel
 
-        text: radioButtonLabelText
         anchors.verticalCenter: parent.verticalCenter
+        anchors.left: radioButton.right
+        anchors.right: parent.right
+        anchors.leftMargin: 16 + radioButton.anchors.rightMargin
+
+        text: radioButtonLabelText
+        lineHeightMode: Text.FixedHeight
+        lineHeight: 21
```

Comment:
Is `lineHeight` set to 21 instead of 22 to accommodate some QML spacing quirkiness? If not, let's use `MZThemes.theme.labelLineHeight`.

Reply:
nope, just me misreading the spec (the label beneath it is 21 🤦‍♂️)